application/android/src/co/theengine/loomplayer/LoomSensors.java

```diff
@@ -192,14 +187,15 @@ public void onSensorChanged(SensorEvent event)
                     ///if the sensor data is unreliable return
                     if(event.accuracy == SensorManager.SENSOR_STATUS_UNRELIABLE)
                     {
-                        return;
+                     //   return;
```

Comment:
I think we can keep this return.. unless it does more damage than it fixes, in which case we should remove the entire conditional.